packages/frontend-2/components/viewer/Controls.vue

```diff
@@ -364,26 +359,60 @@ const {
   diff: { enabled }
 } = useInjectedViewerInterfaceState()
 
+enum ViewerKeyboardActions {
+  ToggleModels,
+  ToggleExplorer,
+  ToggleDiscussions,
+  ToggleMeasurements,
+  ToggleProjection,
+  ToggleSectionBox,
+  ZoomExtentsOrSelection
+}
+
+const map: Record<ViewerKeyboardActions, string> = {
+  [ViewerKeyboardActions.ToggleModels]: 'm',
+  [ViewerKeyboardActions.ToggleExplorer]: 'e',
+  [ViewerKeyboardActions.ToggleDiscussions]: 't',
+  [ViewerKeyboardActions.ToggleMeasurements]: 'r',
+  [ViewerKeyboardActions.ToggleProjection]: 'p',
+  [ViewerKeyboardActions.ToggleSectionBox]: 'b',
+  [ViewerKeyboardActions.ZoomExtentsOrSelection]: ' '
+}
+
+const getKeyboardShortcutTitle = (key: string): string => key
```

Comment:
Currently pointless. You could at least do the `map[key]` bit here, instead of having to duplicate the map[..] reference in all usages below